src/main/java/leetcode/StringToInteger.java

```diff
@@ -10,111 +10,87 @@
  * @author shivam.maharshi
  */
 public class StringToInteger extends TestCase {
+	IntegerValue value = new IntegerValue();
+	public int inputString(String str) {
+		int result = 0;
+		boolean isNeg = false;
+		String input = str.trim();
+		input = reverse(input);
+		input.trim();
+		input = reverse(input);
+		if (input.length() > 0) {
+			char fc = input.charAt(0);
+			if (fc == '-') {
+				isNeg = true;
+			} else if (fc == '+') {
+				// do nothing
+			} else if (getValue(fc) == -1) {
+				return 0;
+			} else {
+				result = getValue(fc);
+			}
+		}
 
-  public int myAtoi(String str) {
-    int res = 0;
-    boolean isNeg = false;
-    String input = str.trim();
-    input = reverse(input);
-    input.trim();
-    input = reverse(input);
-    if (input.length() > 0) {
-      char fc = input.charAt(0);
-      if (fc == '-') {
-        isNeg = true;
-      } else if (fc == '+') {
-        // do nothing
-      } else if (getValue(fc) == -1) {
-        return 0;
-      } else {
-        res = getValue(fc);
-      }
-    }
+		for (int i = 1; i < input.length(); i++) {

```

Comment:
bro, add some comment to this module. or maybe extract out with a good naming? 

the nested 3 level of if-else is not that easy to understand.